virtual-desktop/src/app/window-manager/mvd-window-manager/context-menu/context-menu.component.css

```diff
@@ -65,22 +93,60 @@
 }
 
 .context-menu .multi-menu-item>.submenu {
-  width: 0;
-  height: 0;
-  right: 0;
+  right: 0px;
   top: 0;
   position: absolute;
   display: none;
+  z-index: 1;
 }
 
-.context-menu .multi-menu-item>.submenu>com-rs-mvd-context-menu {
-  margin: -10px;
-  padding: 10px;
+.context-menu .multi-menu-item>.submenu.propagateChildLeft {
+  left: 0px;
 }
 
+.context-menu .multi-menu-item>.submenu>com-rs-mvd-context-menu>.context-menu {
+  left: 0;
+  top: 0;
+}
 
-.context-menu .menu-item:last-child {
-  border-bottom: none;
+.context-menu .multi-menu-item>.submenu>com-rs-mvd-context-menu>.context-menu.propagateChildLeft>ul {
+  position: absolute;
+  right: 100%;
+}
+
+.menuitem-content {
+  display: flex;
+  justify-content: space-between;
+}
+
+.menuitem-content-left {
+  display: flex;
+  margin-right: 15px;
+}
+
+.icon {
+  width: 19px;
+  height: 19px;
+  margin-right: 5px;
+  background-size: contain;
+}
+
+.icon.propagateChildLeft {
+  margin-left: 25px;
+}
+
+.icon-person {
+  background-image: url(../../../../assets/images/contextmenu/person.png);
```

Comment:
Point 6 here... just as you allow a url here, find a way to make it so that people can use a url from anywhere else too.